pylintrc

```diff
@@ -154,7 +154,7 @@ no-space-check =
 
 # Maximum number of lines in a module
 # DEFAULT:  max-module-lines=1000
-max-module-lines=1500
+max-module-lines=2500
```

Comment:
I'm against this change. We should be remembered that `net.py` is too big and eventually split it up.

Reply:
👍